experiments/aws/parallel-cluster/cpu/experiment/README.md

```diff
@@ -198,85 +194,63 @@ Don't forget to save the MiniFE yaml output files that generate in the PWD.
 
 #### Mixbench
 
-This wasn't on the path.
-
 ```bash
-# 49.78 seconds
-time singularity pull docker://ghcr.io/converged-computing/metric-mixbench:libfabric-cpu
-
-# 24 seconds
-time mpirun -np 2 --hostfile ./hostfile.txt /shared/bin/singularity exec /shared/containers/metric-mixbench_libfabric-cpu.sif mixbench-cpu
+cd configs/mixbench/
```

Comment:
Let's make sure to chat with Abhik, Tapasya, about what to do for iterations based on runtimes at different scales. I'm not sure the best way to handle that with a submit vs. interactive run.